src/tasks/publish/index.ts

```diff
@@ -144,9 +144,12 @@ async function publishTask(
 
   // Do sanity checks before compiling the contract or uploading files
   // So users do not have to wait a long time before seeing the config is not okay
-  await assertIpfsApiIsAvailable(ipfsApiUrl)
-
+  if (!rootAccount)
+    throw new BuidlerPluginError(
+      `No account configured. Provide a mnemonic or private key for ${selectedNetwork} in `
```

Comment:
Maybe it's just me, but it seems like we're missing the end of this sentence here

Reply:
Fixed by https://github.com/aragon/buidler-aragon/pull/115